buildscripts/observability-test/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Copyright 2022 gRPC authors.
```

Comment:
2023. In the shell scripts as well.

Reply:
Done. Damn I started working on this since 2022 :)